static/chrome.css

```diff
@@ -0,0 +1,157 @@
+/* Reset */
+ul,ol { list-style: none; }
```

Comment:
Q: what is the static directory? is that new?

Reply:
It's new. I wanted a place for the static markup created for the new chrome, and since there's still a range slider that needs to be replaced I kept it in.